docs/3.0.0-beta.x/community/contribute-with-code.md

```diff
@@ -0,0 +1,103 @@
+# Contribute with Code
+
+We accept all kinds of Pull Requests, including bug fixes, new features, and documentation.
+
+**NOTE:** We encourage you to discuss your proposed Pull Request with maintainers or core team members before submitting new features. In this way, you may get tips and other useful information (e.g., upcoming breaking changes) and you ensure your pull request aligns with the project roadmap.
+
+## Before Submitting a Pull Request
+
+**Before submitting your pull request**, fulfill the following requirements:
+
+- Fork the repository and create your branch from `master`.
+- Run `yarn setup` in the repository root.
+- If you’ve fixed a bug or added code that should be tested, add the tests, and then link the corresponding issue in either your commit or your PR!
+- Ensure the test suites are passing:
+  - `yarn test:unit`
+  - `yarn test:front`
+- Use a linter on your code (`yarn lint`).
+- When you write the Pull Request, please follow the [Pull Request Template](https://github.com/strapi/strapi/blob/master/.github/PULL_REQUEST_TEMPLATE.md).
+
+## Technical Requirements
+
+- You have [Node v10.x.x](https://nodejs.org/en/download/) (LTS) or [Node v12.x.x](https://nodejs.org/en/download/current/) and [Yarn v1.2.0+](https://yarnpkg.com/en/).
```

Comment:
I think we need to create a component for this part are reuse it the documentation in all parts it is stated.

Reply:
@soupette - Yes I will definitely make this a component. I needed @lauriejim to be back.